src/solver/variable/include/antares/solver/variable/adequacy/overallCost.h

```diff
@@ -238,13 +238,20 @@ class OverallCost: public Variable::IVariable<OverallCost<NextT>, NextT, VCardOv
         // Hydro costs : water value and pumping
         pValuesForTheCurrentYear[numSpace].hour[state.hourInTheYear]
           += state.problemeHebdo->CaracteristiquesHydrauliques[state.area->index]
-               .WeeklyWaterValueStateRegular
-             * (state.hourlyResults->TurbinageHoraire[state.hourInTheWeek]
-                - area->hydro.pumpingEfficiency
-                    * state.hourlyResults->PompageHoraire[state.hourInTheWeek]);
+                 .WeeklyWaterValueStateRegular
```

Comment:
suggestion: This is become a bit complex. Maybe split in different variables/functions to improve readability and comprehension.